Before: 9b49c54dd5e08528fa5b3f21c95fb89aa3e82057afa908e8853fd10704f95181
After: f7e940a0f042ad6c084bd4c4dc4e4c3dd83b74b06fb8970871ff4d5c938ccc7f
Patch: a3591c9c331d8cf931c62101f4d6b92f71299b2321d8313dc10c55739e780a61
--- %name%.csv.tt	2012-09-11 13:27:33.354634088 +1000
+++ /dev/null	2012-09-03 14:23:42.005077313 +1000
@@ -1,2 +0,0 @@
-NAME,AGE
-<%= name %>,<%= age %>
--- %name%.txt.tt	2012-09-11 13:27:33.354634088 +1000
+++ %name%.txt.tt	2012-09-11 13:27:33.321300555 +1000
@@ -1,3 +1,3 @@
 My name is <%= name %>.
 I'm <%= age %> years old.
-Project name <%= project_name %>
+
--- corvid-features/template2.rb	2012-09-11 13:27:33.354634088 +1000
+++ corvid-features/template2.rb	2012-09-11 13:27:33.321300555 +1000
@@ -1,6 +1,5 @@
 install {
   template2 '%name%.txt.tt'
-  template2 '%name%.csv.tt'
 }
 
 def name; 'George' end
